docs/k8s/operator-architecture.md

```diff
@@ -50,9 +66,14 @@ flowchart LR
 [Documentation][kb-operator-l3-ingress]
 
 The user deploys an app to the default namespace, and creates a normal Service
-that selects the app's pods. Add the annotation `tailscale.com/expose: "true"`
-to the Service, and the operator will create an ingress proxy that allows
-devices anywhere on the tailnet to access the Service.
+that selects the app's Pods. Either add the annotation
+`tailscale.com/expose: "true"` or specify `.spec.type` as `Loadbalancer` and
+`.spec.loadBalancerClass` as `tailscale`. The operator will create an ingress
+proxy that allows devices anywhere on the tailnet to access the Service.
+
+The proxy Pod uses `iptables` or `nftables` rules to DNAT traffic bound for the
+proxy's tailnet IP to the Service's internal Cluster IP instead. This usually
+means traffic coming in on `tailscale0` is forwarded to the `eth0` interface.
```

Comment:
Suggestion:
```suggestion
The proxy Pod uses `iptables` or `nftables` rules to DNAT traffic bound for the
proxy's tailnet IP to the Service's internal Cluster IP instead. 
```
(altough in practice usually true, it doesn't really add anything to the previous sentence and seems a bit confusing)